src/bsk_rl/data/unique_image_data.py

```diff
@@ -142,7 +142,13 @@ def __init__(
     def create_data_store(self, satellite: "Satellite") -> None:
         """Override the access filter in addition to creating the data store."""
         super().create_data_store(satellite)
-        satellite.get_access_filter = lambda: satellite.data_store.data.imaged
+
+        def unique_target_filter(opportunity):
```

Comment:
Here, the idea is to change from passing the list of unavailable targets to passing the list of available targets.